packages/instant/src/components/timed_progress_bar.tsx

```diff
@@ -89,11 +91,23 @@ export const Progress =
     }
 `;
 
+export const ProgressBackground = styled.div`
+    && {
+        background-color: ${props => props.theme[ColorOption.primaryColor]};
+        border-radius: ${progressRadius};
+        height: ${progressHeight};
+        position: absolute;
+        opacity: 0.1;
+        width: 100%;
+    }
+`;
+
 export interface ProgressBarProps extends ProgressProps {}
 
 export const ProgressBar: React.ComponentType<ProgressBarProps & React.ClassAttributes<{}>> = React.forwardRef(
     (props, ref) => (
-        <Container width="100%" backgroundColor={ColorOption.lightGrey} borderRadius="6px">
```

Comment:
Why don't we just transform the color and set it as the background color of this container instead of creating a new div?

Reply:
That is what I wanted to do, but couldn't find a good way.  There may be an approach I'm missing tho, lemme know:
- If I set an opacity on this container (line 96), it affects the opacity of the moving progress bar too
- There doesn't seem to be a good way to just set the background color opacity to a value
- The primary color could be a hex value and not rgba, and I figured I'd have to bring in additional dependency to generate a new value w/ a specificed opacity

Reply:
https://polished.js.org/docs/#opacify

Reply:
https://polished.js.org/docs/#transparentize rather

Reply:
Done!